extensions/blocks/map/map.php

```diff
@@ -25,7 +25,83 @@
 function jetpack_map_block_load_assets( $attr, $content ) {
 	$api_key = Jetpack_Options::get_option( 'mapbox_api_key' );
 
+	if ( class_exists( 'Jetpack_AMP_Support' ) && Jetpack_AMP_Support::is_amp_request() ) {
+		global $wp, $map_block_counter;
+		if ( ! $map_block_counter ) {
```

Comment:
I think this should check if `! isset( $map_block_counter )`:

```suggestion
		if ( ! isset( $map_block_counter[ get_the_ID() ] ) ) {
```

Reply:
I missed a line - `$map_block_counter` isn't meant to be an object, it's a running total of how many Map blocks are in the current page.  Corrected in https://github.com/Automattic/jetpack/pull/13405/commits/f05ee9d924590836a349641313e93454c5bced24